.gitmodules

```diff
@@ -37,7 +37,9 @@
 	url = https://github.com/iree-org/stablehlo.git
 [submodule "third_party/torch-mlir"]
 	path = third_party/torch-mlir
-	url = https://github.com/iree-org/torch-mlir.git
+	url = https://github.com/rohan-tan-bhowmik/torch-mlir.git
 [submodule "third_party/hip-build-deps"]
 	path = third_party/hip-build-deps
 	url = https://github.com/iree-org/hip-build-deps.git
+[submodule "third_party/torch-mlir/submodule"]
+	url = https://github.com/rohan-tan-bhowmik/torch-mlir.git
```

Comment:
I assume these submodule changes are so you can test using IREE's CI? Can you convert this PR to a draft while these sorts of changes are included?

Reply:
yes, this PR depends on https://github.com/llvm/torch-mlir/pull/3690

Reply:
but would like to prefetch the comments and refactorings :)

Reply:
PR has been applied!